Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMHTMLFrameElement.h

```diff
@@ -246,28 +246,6 @@ webkit_dom_html_frame_element_get_content_document(WebKitDOMHTMLFrameElement* se
 WEBKIT_DEPRECATED WebKitDOMDOMWindow*
 webkit_dom_html_frame_element_get_content_window(WebKitDOMHTMLFrameElement* self);
 
-/**
- * webkit_dom_html_frame_element_get_width:
- * @self: A #WebKitDOMHTMLFrameElement
- *
- * Returns: A #glong
- *
- * Deprecated: 2.22: Use JavaScriptCore API instead
-**/
-WEBKIT_DEPRECATED glong
-webkit_dom_html_frame_element_get_width(WebKitDOMHTMLFrameElement* self);
-
-/**
- * webkit_dom_html_frame_element_get_height:
- * @self: A #WebKitDOMHTMLFrameElement
- *
- * Returns: A #glong
- *
- * Deprecated: 2.22: Use JavaScriptCore API instead
-**/
-WEBKIT_DEPRECATED glong
-webkit_dom_html_frame_element_get_height(WebKitDOMHTMLFrameElement* self);
```

Comment:
Then you would leave the header unchanged.

Reply:
Actually I suppose we should change `Returns: A #glong` to `Returns: 0` for good measure.

Reply:
Done.